styles/components/Tabs.module.scss

```diff
@@ -0,0 +1,26 @@
+@use "../vars.module.scss";
+
+.tabs {
+  width: 100%;
```

Comment:
May I  suggest we make the buttons a little higher? I think it improves readability.

```suggestion
  width: 100%;
  height: 40px;
```

| Before: <br /> ![image](https://user-images.githubusercontent.com/37463445/179966680-d7122d9b-9651-4d83-b680-d42da8c0b59b.png) | After: <br /> ![image](https://user-images.githubusercontent.com/37463445/179966924-38cf096c-1684-434f-9f63-d9347d24700d.png) |
|-- |-- |